bulbs/videos/templates/videos/series-page.html

```diff
@@ -15,26 +15,65 @@
  <p>Watch The Latest Episode</p>
 </section>
 
-<figure class="video-container">
```

Comment:
Did you check these edits out on AVC? Just want to make sure they don't break any existing views